staging/src/k8s.io/apimachinery/pkg/util/strategicpatch/patch_test.go

```diff
@@ -648,1345 +826,3137 @@ testCases:
       nonMergingIntList:
         - 2
         - 3
-  - description: merge lists of scalars
-    original:
-      mergingIntList:
-        - 1
-        - 2
-    twoWay:
-      mergingIntList:
-        - 3
-    modified:
-      mergingIntList:
-        - 1
-        - 2
-        - 3
-    current:
-      mergingIntList:
-        - 1
-        - 2
-        - 4
-    threeWay:
-      mergingIntList:
-        - 3
-    result:
-      mergingIntList:
-        - 1
-        - 2
-        - 3
-        - 4
-  - description: merge lists of maps
+  - description: delete all maps from merging list
```

Comment:
the diff in the rest of this file is extremely difficult to review... were the changes simple moves/refactors or were there changes made?